packages/contracts-bedrock/src/L2/L2ToL2CrossDomainMessenger.sol

```diff
@@ -114,6 +114,16 @@ contract L2ToL2CrossDomainMessenger is IL2ToL2CrossDomainMessenger, ISemver, Tra
         }
     }
 
+    /// @notice Retrieves the context of the current cross domain message. If not entered, reverts.
+    /// @return sender_ Address of the sender of the current cross domain message.
+    /// @return source_ Chain ID of the source of the current cross domain message.
+    function crossDomainMessageContext() external view onlyEntered returns (address sender_, uint256 source_) {
```

Comment:
Should we just remove `crossDomainMessageSender()` and `crossDomainMessageSource()` in order to simplify the interface? I see that they are no longer used by the `SuperchainTokenBridge`.

Reply:
Those functions were already in place before we modified the contracts so we are not totally sure what was the thought behind having it separatly. But for example, in a previous implementation of this contract it was useful because we only used the sender. The need of the source it was decided and added later.

We don’t have a strong preference here, just wanted to keep in mind that this contract could have use cases beyond the current one.